components/BugZap/BugZapGame.js

```diff
@@ -248,10 +269,23 @@ class BugZapGame extends React.Component {
       });
       this.characterHopOn();
     }
-    // this.characterHopOff();
+    else if (characterUID === 'character' && this.readyToCelebrate) {
+      index = [5,6,6,6,0]
```

Comment:
If possible use character.animationIndex('CELEBRATE'); rather then for this commit, I would change over to use newer character js things like character.size.width/height, character.animationIndex etc.